src/features/environment.js

```diff
@@ -38,10 +41,16 @@ export const EnvironmentMixin = (ModelViewerElement) => {
         ...super.properties,
         backgroundImage: {type: String, attribute: 'background-image'},
         backgroundColor: {type: String, attribute: 'background-color'},
-        experimentalPmrem: {type: Boolean, attribute: 'experimental-pmrem'}
+        experimentalPmrem: {type: Boolean, attribute: 'experimental-pmrem'},
+        shadowStrength: {type: Number, attribute: 'shadow-strength'}
```

Comment:
I think `shadow-intensity` may be more in line with how shadow "strength" is normally defined

Reply:
I am planning to add some "intensity" configurations that control lighting. Since shadow is the absence of light, intensity seemed like a weird word to use.

Reply:
Changed to intensity, as @johnpallett points out we may want a complimentary value in the future called "softness" which seems weird to pair with "strength." Then again, must one always be either soft or strong, or is this just a bias instilled in me by social prejudice?